aspect/testing/tests/src/com/google/idea/blaze/aspect/integration/BUILD

```diff
@@ -15,8 +17,11 @@ java_binary(
     ],
 )
 
-bazel_integration_tests(
+bazel_integration_test(
```

Comment:
did this disable the tests against older versions of bazel? I do not see it in the test result https://buildkite.com/bazel/intellij-plugin-aspect/builds/24766#01954c26-b10e-4a63-8d88-38d1b7f04258

Reply:
Yes it does. Because the integration tests do not support templating and therefore it is not possible to target multiple bazel versions with these changes.